### DISCUSSION TITLE
Headers for Tables

### DISCUSSION THREAD

**Discussion by jpobley** [ueberdosis/tiptap#2411] [General Questions] [answered]
I noticed that the TipTap tables extension is using the deprecated [`toggleHeaderColumn`](https://github.com/ueberdosis/tiptap/blob/9e48f8c23960198b166b23804c094f2ddccc3927/packages/extension-table/src/table.ts#L160) and [`toggleHeaderRow`](https://github.com/ueberdosis/tiptap/blob/9e48f8c23960198b166b23804c094f2ddccc3927/packages/extension-table/src/table.ts#L163) commands from `prosemirror-tables` instead of the preferred [`toggleHeader`](https://github.com/ProseMirror/prosemirror-tables/blob/master/src/commands.js#L394) command.

The deprecated functions toggle header cells in whichever rows/columns are selected, while the preferred command toggles header cells in either the first row or the first column depending on the argument provided.

 Was this a conscious decision or should a PR be opened to change it?

**Comment by hanspagel** [comment, marked as answer]
Oops! Yes, a PR would be amazing!

**Reply by jpobley** [reply]
https://github.com/ueberdosis/tiptap/pull/2412
